Source/Particles/WarpXParticleContainer.cpp

```diff
@@ -1517,6 +1517,10 @@ WarpXParticleContainer::PushX (int lev, amrex::Real dt)
 // without runtime component).
 void WarpXParticleContainer::defineAllParticleTiles () noexcept
 {
+    // Call the parent class's method
+    NamedComponentParticleContainer<amrex::DefaultAllocator>::defineAllParticleTiles();
+
+    // Resize the tmp_particle_data (no present in parent class)
     tmp_particle_data.resize(finestLevel()+1);
```

Comment:
@atmyers Out of curiosity, do you remember what the `tmp_particle_data` is used for?

Reply:
It's used in the back transformed diagnostics, IIRC to store the old particle positions.